lookout/style/format/descriptions.py

```diff
@@ -266,9 +266,14 @@ def get_change_description(vnode: VirtualNode, feature_extractor: FeatureExtract
     old_label = class_representations[feature_extractor.class_sequences_to_labels[vnode.y_old]]
     new_label = class_representations[feature_extractor.class_sequences_to_labels[vnode.y]]
     if vnode.y[0] == CLASS_INDEX[CLS_NOOP]:
-        return "%s at column %d should be removed." % (old_label, column)
+        if CLASS_INDEX[CLS_NEWLINE] in vnode.y_old:
+            return "Redundant line break. Please concatenate with previous line."
```

Comment:
```suggestion
            return "Redundant line break. Please concatenate with the following line."
```

previous => following because we place our suggestion on top, not on bottom

Reply:
I changed it. 
Here the explanation https://github.com/src-d/style-analyzer/pull/573/files#r252780292 and the example https://github.com/zurk/redux/pull/6#discussion_r252800051 the same for replaces with NOOP

Are you fine with that?

Reply:
I am not able to think it properly but if you are sure then fine - just include "the' article

Reply:
good!

Reply:
*the* previous line